src/helm/benchmark/adaptation/adapter_spec.py

```diff
@@ -87,3 +87,7 @@ class AdapterSpec:
 
     # Random string (used concretely to bypass cache / see diverse results)
     random: Optional[str] = None
+
+    # If true, for instances with multiple correct reference, the gold answer should be considered
+    # to be all of the correct references rather than any of the correct references.
+    multi_label: Optional[bool] = None
```

Comment:
What's the difference between false and None?

Reply:
I just made this a `bool`... I was doing `Optional[bool]` for JSON reverse compatibility, but I don't think this matters too much.